contracts/lib/LibBalances.sol

```diff
@@ -110,11 +110,22 @@ library Balances {
         return liquidationGasCost + minimumMarginWithoutGasCost;
     }
 
+    function fillAmount(
```

Comment:
I reckon fillAmount should just take in both orders and both an aFilled and bFilled and return. That way we can call it once from the perp

Reply:
Consult d0486d1